R/03-rgpt3.R

```diff
@@ -0,0 +1,263 @@
+#' Set up the authentication with your API key
+#'
+#' @description Access to GPT-3's functions requires an API key that you obtain
+#'   from [https://openai.com/api/](https://openai.com/api/).
+#'   `gpt3_authenticate()` looks for your API key in a file that you provide the
+#'   path to and ensures you can connect to the models. `gpt3_endsession()`
+#'   overwrites your API key _for this session_ (it is recommended that you run
+#'   this when you are done). `check_apikey_form()` is a simple check if any
+#'   information has been provided at all.
+#' @param path The file path to the API key
+#' @return A confirmation message
+#' @details The easiest way to store you API key is in a `.txt` file with _only_
+#'   the API key in it (without quotation marks or other common string
+#'   indicators). `gpt3_authenticate()` reads the single file you point it to
+#'   and retrieves the content as authentication key for all requests.
+#'
+#' @export
+gpt3_authenticate = function(path){
+  apikey_ = readLines(path)
+  pkg.env$api_key = apikey_
```

Comment:
pkg.env is not defined. Can you check how to set it?

Reply:
Yes, i missed that. Its now defined in the `00-config.R` file